README.md

```diff
@@ -37,6 +37,10 @@ Use `benchopt run -h` for more details about these options, or visit https://ben
 
 ### List of optimization problems available
 
+- `ols`: aka ordinary least-squares. This consists in solving the following program:
+
+  ```min_w 1 / 2 * ||y - Xw||^2_2```
```

Comment:
you version is too old. This file is gone in current master. REvert this commit and do:

git pull origin master 

in this branch

Reply:
I hope I did it right. Let me know.